### DISCUSSION TITLE
Request: OpenUtau for Android

### DISCUSSION THREAD

**Discussion by Articuno227** [stakira/OpenUtau#597] [Ideas]
There's not much to say about it.

Android version of synthesizer such as OpenUtau would bring much more people into community, who would like to make tons of work like original songs or covers. Y'know not so many people have access to the pc, including frequent access as well as any of it.

Fully-functioning synthesizer for Android will change vocasynth community forever, and it would be a good change.

I believe there won't be performance issues, since phones nowadays are better for anything then trashy 10 years old PCs.

Also it would be a great solution as synthesizer in your pocket.

Thanks for letting me sharing this idea! I hope mobile synthesizers will exist somewhen, and OpenUtau is perfect for this.

**Comment by oxygen-dioxide** [comment]
- It's theoretically possible to port OpenUtau to mobile platforms. Since OpenUtau is written in [dotnet](https://dotnet.microsoft.com), and there are several dotnet cross-platform UI frameworks for mobile, such as [Uno Platform](https://platform.uno/) and [Maui](https://dotnet.microsoft.com/en-us/apps/maui).
- Mobile operating systems and user experiences are different from desktop's, which requires a fully UI-redesign and rewrite. OpenUtau is an open-source software. Unlike commercial softwares, OpenUtau is developed by community in their spare time. Development bandwith is limited. However, since the source code is public, anyone can implement their ideas.
- There are still many limitations. Android doesn't support exe, which means that you can't use third-party resamplers or wavtools on Android, such as Moresampler. NNSVS, originally written in python, would need a fully C# port before running on Android. Only Worldline-R and vogen would be supported.

**Comment by OriginStar-Xpp** [comment]
Just as the same reason，I also wanna OpenUTAU for iOS or iPadOS

**Reply by MehurPlayz** [reply]
YEAHHH

**Comment by bekotoonz** [comment]
robot singers+phone port=skibidi toilet song made by a gen alpha kid【UTAU COVER】

**Reply by trustypark** [reply]
oh god that's horrifying

**Reply by keXa2008** [reply]
worth it

**Reply by NulledInstance** [reply]
i higly doubt they can even use openutau

**Comment by oxygen-dioxide** [comment]
A developer ported OpenUtau to mobile: https://github.com/vocoder712/OpenUtauMobile
